.gitignore

```diff
@@ -148,8 +148,5 @@ yarn.lock
 .vscode
 .idea
 
-#tap files
-.tap/
-
```

Comment:
Might be worth keeping this around for a bit just in case for people who ran tests